app/models/master_file.rb

```diff
@@ -314,11 +314,11 @@ def update_progress_on_success!(encode)
     # We can get the proper aspect ratio from the transcoded files, so we set the master file off the 
     # encode output.
     if is_video?
-      high_output = Array(encode.output).select { |out| out.label.include?("high") }.first
+      high_output = Array(encode.output).select { |out| out.label&.include?("high") }.first
       self.display_aspect_ratio = (high_output.width.to_f / high_output.height.to_f).to_s
     end
 
-    outputs = Array(encode.output).collect do |output|
+    outputs = Array(encode.output).reject { |output| output.subtitles.nil? }.collect do |output|
```

Comment:
I think it would be good to have a clearer test that only captures the named outputs requested when the encode is created.  Maybe something like this is sufficient?
```suggestion
    outputs = Array(encode.output).select { |output| ["high", "medium", "low"].include?(output.label) }.collect do |output|
```

Reply:
As I've been working on getting other stuff to work with the spec test, I don't think that this test will work consistently enough. 

I'm not sure that outputs are guaranteed to be named "high", "low", etc. In `config/ffmpeg_presets.yml`, the label for outputs is a configurable option. I doubt that people would change that but they could. Also, do we know if files processed through the S3 adapters are always labelled "high", etc? Can we guarantee that is consistent?

Maybe a better test for now would be to reject `output.format == 'vtt'` like we filter off of in watched_encode? Or would there be an easy way to get the actual values the encode was created with and filter off those?

Reply:
I think getting the output labels from the original encode request would be ideal but rejecting `output.format == 'vtt'` seems good enough for now.